themes/gruvbox-dark-hard.json

```diff
@@ -1062,6 +988,13 @@
     "editorError.foreground": "#cc241d",
     "editorWarning.foreground": "#d79921",
     "editorInfo.foreground": "#458588",
+    // EDITOR - BRACKET PAIR COLORIZATION
+    "editorBracketHighlight.foreground1": "#b16286",
+    "editorBracketHighlight.foreground2": "#458588",
+    "editorBracketHighlight.foreground3": "#98971a",
+    "editorBracketHighlight.foreground4": "#d79921",
+    "editorBracketHighlight.foreground6": "#d65d0e",
```

Comment:
it seems we're missing `foreground6` ?

Reply:
Oh, it appears that we're indeed missing fg5. 
Will correct & re-commit. Thanks!